rma/models/stock_warehouse.py

```diff
@@ -69,7 +69,6 @@ def write(self, vals):
                                 "usage": "internal",
                                 "location_id": wh.view_location_id.id,
                                 "company_id": wh.company_id.id,
-                                "return_location": True,
```

Comment:
Why this change? The RMA location isn't by definition a return location?

Reply:
Field has been removed in 18.0

Reply:
Makes sense!